src/control/drpc/rpc.go

```diff
@@ -23,6 +23,7 @@
 package drpc
 
 import (
+	"github.com/daos-stack/daos/src/control/log"
```

Comment:
As this is an internal import, it should be in its own group after the 3rd-party imports.

Reply:
Im not sure why but this is the default behavior for the editor. I type log.Debugf or something like that and it asks which import and places it. I'll move it.

Reply:
The editor doesn't really know about 3rd-party vs internal imports. This is a convention rather than a rule, so it's up to us to keep things organized.